src/content/docs/en/pages/guides/marketplace/templates/stencil-pwa-boilerplate.mdx

```diff
@@ -0,0 +1,89 @@
+---
+title: How to deploy the Stencil PWA Boilerplate
+description: This template contains the configurations for deploying a Progressive Web Application (PWA) using Stencil.
+meta_tags: templates, guides, Azion Marketplace, Stencil
+namespace: docs_guides_stencil_pwa_boilerplate
+permalink: /documentation/products/guides/stencil-pwa-boilerplate/
+---
+
+import Tag from 'primevue/tag';
+
+<Tag severity="info" client:only="vue">
+Preview
+</Tag>
+
+The **Stencil PWA Boilerplate** template contains the configurations to quickly build a [Stencil](https://stenciljs.com/docs/introduction) based [Ionic PWA](https://ionic.io/pwa) (Progressive Web Application), a toolchain for building fast, reusable UI components and static sites.
+
+The deployment of this template creates a GitHub repository containing your project, as well as an edge application and a domain to facilitate your access and management through Azion Edge Platform.
+
+This template uses Stencil version `4.22.2`.
+
+## Requirements
+
+- A [GitHub account](https://github.com/signup) to connect with Azion and create your new repository.
+    - Every push will be deployed automatically to this repository to keep your project updated.
+- This template uses [Application Accelerator](/en/documentation/products/build/edge-application/application-accelerator/), [Edge Functions](/en/documentation/products/build/edge-application/edge-functions/), and [Edge Cache](https://www.azion.com/en/documentation/products/build/edge-application/edge-cache/), and it could generate usage-related costs. Check the [pricing page](https://www.azion.com/en/pricing/) for more information.
```

Comment:
```suggestion
- This template uses [Application Accelerator](/en/documentation/products/build/edge-application/application-accelerator/), [Edge Functions](/en/documentation/products/build/edge-application/edge-functions/), and [Edge Cache](/en/documentation/products/build/edge-application/edge-cache/), and it could generate usage-related costs. Check the [pricing page](https://www.azion.com/en/pricing/) for more information.
```